servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/reporter/HttpReporter.java

```diff
@@ -246,11 +246,11 @@ public Builder maxConcurrentReports(final int maxConcurrentReports) {
          * @param maxBatchDuration {@link Duration} of time to wait for {@code batchSizeHint} spans in a batch.
          * @return {@code this}.
          */
-        public Builder batchSpans(final int batchSizeHint, final Duration maxBatchDuration) {
```

Comment:
can we first deprecate the `batchSpans(int, Duration)` method, and add a new method `spansBatchingEnabled(int, Duration)` to reduce impacts of API breakage?